rapid_response_xblock/models.py

```diff
@@ -0,0 +1,42 @@
+"""
+Models for storing rapid response grades
+"""
+from __future__ import unicode_literals
+
+from django.conf import settings
+from django.db import models
+from django.utils.encoding import python_2_unicode_compatible
+from jsonfield import JSONField
+from model_utils.models import TimeStampedModel
+from opaque_keys.edx.django.models import (
+    CourseKeyField,
+    UsageKeyField,
+)
+
+
+@python_2_unicode_compatible
+class ProblemCheckRapidResponse(TimeStampedModel):
+    """
+    Keep track of events relating to rapid responses
+    """
+    user = models.ForeignKey(
+        settings.AUTH_USER_MODEL,
+        on_delete=models.SET_NULL,
+        null=True,
+        db_index=True,
+    )
+    problem_id = UsageKeyField(db_index=True, max_length=255)
+    course_id = CourseKeyField(db_index=True, max_length=255)
+    answer_id = models.CharField(null=True, max_length=255)
+    answer_text = models.CharField(null=True, max_length=4096)
+    event = JSONField()
```

Comment:
Yes, this field is only there just in case. We should be pulling the information out into the other fields, it is not efficient to query these JSONFields.